create_dockerfile.sh

```diff
@@ -1,13 +1,16 @@
-docker run --rm kaczmarj/neurodocker:v0.3.2 generate -b neurodebian:stretch-non-free -p apt \
-    --install fsl \
+docker run --rm kaczmarj/neurodocker:0.4.3 generate docker -b neurodebian:stretch-non-free -p apt \
+    --install fsl gcc g++ software-properties-common \
     --user=neuro \
     --add-to-entrypoint "source /etc/fsl/fsl.sh" \
-    --miniconda env_name=nimare \
+    --miniconda create_env=nimare \
     miniconda_version=4.3.31 \
-    conda_install="python=3.6 jupyter jupyterlab jupyter_contrib_nbextensions
-                   matplotlib scikit-learn seaborn numpy scipy pandas
-                   statsmodels nibabel nipype" \
-    pip_install="nilearn pybids duecredit pymc3" \
+    conda_install="python=3.6 jupyter jupyterlab jupyter_contrib_nbextensions seaborn" \
+    pip_install="https://github.com/neurostuff/NiMARE/tarball/master" \
```

Comment:
```suggestion
    pip_install="/opt/NiMARE/" \
```
to install the code we copied in from above.